packages/react/src/FAQ/FAQ.features.stories.tsx

```diff
@@ -120,8 +120,8 @@ export const AllOpen: StoryFn<typeof FAQ> = () => {
   return (
     <Container>
       <FAQ>
-        <FAQ.Heading>Frequently asked&nbsp;questions</FAQ.Heading>
-        <>
+        <FAQ.Heading id="all-open-faq">Frequently asked&nbsp;questions</FAQ.Heading>
```

Comment:
How would you feel about making these props required and leaving information in the documentation? We could do this dynamically but it would require a second loop of the children to first grab the id if a user specifies one and then set the id on the group. Of course, if they didn't supply one then this second loop wouldn't be needed but I was thinking we would want to support them passing their own id if they wanted to access that component using an id query.